pkgs/servers/web-apps/wordpress/wordpress-core.nix

```diff
@@ -0,0 +1,10 @@
+{ wordpress }:
+
+wordpress.overrideAttrs (oldAttrs: {
+  pname = "wordpress-core";
+
+  installPhase = oldAttrs.installPhase + ''
+    rm -r $out/share/wordpress/wp-content/plugins/*
+    rm -r $out/share/wordpress/wp-content/themes/*
+  '';
```

Comment:
This works, but I wonder if using `override` and a flag argument is the more sound approach. Then instead of deleting them you could just not install them in the first place.

```
wordpress-core = wordpress.override { installDefaultThemes = false; installDefaultPlugins = false; };
```